includes/openhab2-override.conf

```diff
@@ -1,5 +1,3 @@
 [Unit]
 Description=openHAB2 instance, reachable at http://%H:8080
-Wants=zram-config.service
-Wants=frontail.service
-
+Wants=zram-config.service frontail.service
```

Comment:
are you sure this works? I looked around the Inet when I've added it but did not find a confirmation so went the safe route (it's also better to have separate lines should we decide to dynamically generate/change this one day via script)

Reply:
I looked at the MAN page and it said it works.

Reply:
Please validate first.

Reply:
you mean you did validate?

Reply:
Not yet. I will this afternoon.

Reply:
It works, no issues for me.